hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordReader.java

```diff
@@ -229,6 +230,10 @@ protected synchronized void scanInternal(Option<KeySpec> keySpecOpt) {
         HoodieLogBlock logBlock = logFormatReaderWrapper.next();
         final String instantTime = logBlock.getLogBlockHeader().get(INSTANT_TIME);
         totalLogBlocks.incrementAndGet();
+        if (logBlock.getBlockType() == CDC_DATA_BLOCK) {
+          // hit a cdc block, just skip.
+          continue;
```

Comment:
get instant time above should be skipped if skipping here. also, should totalLogBlocks increment? pls confirm

Reply:
according to the following logic, even the block doesn't be read, `totalLogBlocks` also should increment.
![image](https://user-images.githubusercontent.com/10036681/187384289-bf2323b8-2b22-45b9-8c49-13c5eba9145e.png)